Library/Homebrew/formula.rb

```diff
@@ -872,39 +872,14 @@ def with_logging(log_type)
     @active_log_type = old_log_type
   end
 
-  # This method can be overridden to provide a plist.
-  # @see https://www.unix.com/man-page/all/5/plist/ Apple's plist(5) man page
-  # <pre>def plist; <<~EOS
-  #  <?xml version="1.0" encoding="UTF-8"?>
-  #  <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-  #  <plist version="1.0">
-  #  <dict>
-  #    <key>Label</key>
-  #      <string>#{plist_name}</string>
-  #    <key>ProgramArguments</key>
-  #    <array>
-  #      <string>#{opt_bin}/example</string>
-  #      <string>--do-this</string>
-  #    </array>
-  #    <key>RunAtLoad</key>
-  #    <true/>
-  #    <key>KeepAlive</key>
-  #    <true/>
-  #    <key>StandardErrorPath</key>
-  #    <string>/dev/null</string>
-  #    <key>StandardOutPath</key>
-  #    <string>/dev/null</string>
-  #  </dict>
-  #  </plist>
-  #  EOS
-  # end</pre>
-  def plist
-    nil
+  # The generated launchd {.plist} service name.
+  def self.plist_name
+    "homebrew.mxcl." + name
   end
 
-  # The generated launchd {.plist} service name.
+  # @private
   def plist_name
-    "homebrew.mxcl." + name
+    self.class.plist_name
```

Comment:
This should actually stay as is.

Reply:
So then we just have `"homebrew.mxcl." + name` duplicated in both `def plist_name` instance, and `def self.plist_name` class methods? (`name` of course being still inacessible from the later… and I don't think it's possible to do the reverse, which is to say forward the `def self.plist_name` class method to this instance method 🤦‍♂️)